apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/block_test.exs

```diff
@@ -0,0 +1,78 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherRPC.Web.Controller.BlockTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: false
+  use OMG.Fixtures
+  use OMG.WatcherInfo.Fixtures
+  use OMG.Watcher.Fixtures
+
+  alias Support.WatcherHelper
+
+  describe "get_block\2" do
+    @tag fixtures: [:initial_blocks]
+    test "/block.get rejects parameter of wrong type" do
+      string_blknum = "1000"
+      %{"data" => data} = WatcherHelper.rpc_call("block.get", %{blknum: string_blknum}, 200)
+
+      expected = %{
+        "code" => "operation:bad_request",
+        "description" => "Parameters required by this operation are missing or incorrect.",
+        "messages" => %{"validation_error" => %{"parameter" => "blknum", "validator" => ":integer"}},
+        "object" => "error"
+      }
+
+      assert data = expected
+    end
+
+    @tag fixtures: [:initial_blocks]
+    test "/block.get endpoint rejects request without parameters" do
+      missing_param = %{}
+      %{"data" => data} = WatcherHelper.rpc_call("block.get", missing_param, 200)
+
+      expected = %{
+        "code" => "operation:bad_request",
+        "messages" => %{"validation_error" => %{"parameter" => "blknum", "validator" => ":integer"}},
+        "object" => "error"
+      }
+
+      assert data = expected
+    end
+
+    @tag fixtures: [:initial_blocks]
+    test "/block.get returns expected error if block not found" do
+      non_existent_block = 5000
+      %{"data" => data} = WatcherHelper.rpc_call("block.get", %{blknum: non_existent_block}, 200)
+
+      expected = %{
+        "code" => "get_block:block_not_found",
+        "description" => nil,
+        "object" => "error"
+      }
+
+      assert data = expected
+    end
+
+    @tag fixtures: [:initial_blocks]
+    test "/block.get returns correct block if existent" do
```

Comment:
Can we move the happy path test to the top?

Reply:
Done 75a2b50.